src/services/misesEnRelation/controllers/validationRecrutementContrat.ts

```diff
@@ -8,7 +8,10 @@ import { action, ressource } from '../../../helpers/accessControl/accessList';
 import service from '../../../helpers/services';
 import { getCoselec } from '../../../utils';
 import { IUser } from '../../../ts/interfaces/db.interfaces';
-import { countConseillersRecrutees } from '../misesEnRelation.repository';
+import {
```

Comment:
ligne où il y a ` if (miseEnRelationVerif.conseillerObj.email.includes(gandi.domain)) {` ça pourrait bloquer dans le cas fin naturelle, à check si Morgan le laisse ou l'enlève aux préalable  .

Reply:
ligne 360 `$in: ['finalisee_rupture', 'terminee', 'nouvelle_rupture'],` 
normalement juste `$in: ['finalisee_rupture', 'terminee']`

Reply:
c'est dans la PR de Morgan

Reply:
Pour la partie recrutement @Ornella452 bonne remarque on va avoir un souci pour ceux qui ont conservé leur espace.... Ou alors il faut que côté @MorganP-projects si recruté ailleurs entre temps on garde juste MM&Gandi. Son compte coop va redevenir candidat....

Reply:
Oui  blocage pour 1 cas précis (recruter par une autre SA avant les 2 mois atteint)

Reply:
Ok je me note ca, ce sera dans l'autre PR de @MorganP-projects

Reply:
Vu une modif est à prévoir ici